src/main/java/club/sk1er/hytilities/handlers/chat/ChatHandler.java

```diff
@@ -137,8 +133,8 @@ public String handleSentMessage(@NotNull String message) {
     }
 
     /**
-     * Fixes styling when overwriting a message.
-     * Store the component siblings before applying any changes to the original component.
+     * Fixes styling when modifying a message's events.
```

Comment:
why are you overwriting https://github.com/Sk1erLLC/Hytilities/commit/1bae687b803b9fad6532383e311cebaad1f9c585